src/service/viewport/viewport-binding-ios-embed-wrapper.js

```diff
@@ -230,13 +244,17 @@ export class ViewportBindingIosEmbedWrapper_ {
 
   /** @override */
   getLayoutRect(el, opt_scrollLeft, opt_scrollTop) {
+    const b = el./*OK*/getBoundingClientRect();
```

Comment:
But the future: do you plan to kill `getLayoutRect` from this class?

Reply:
Yes. Everything should instead call into Layers.